x/globalfee/types/params.go

```diff
@@ -9,7 +11,8 @@ import (
 // ParamStoreKeyMinGasPrices store key
 var ParamStoreKeyMinGasPrices = []byte("MinimumGasPricesParam")
 
-// DefaultParams returns default wasm parameters
+// todo set DefaultGlobalFee
```

Comment:
i dont understand. the if we. do not set anything for global fee , it is empty. we want default to be 0uatom right ? so that we can constrain paying fees denom.